### PR TITLE
Add multiple tier 1 packages

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: packages/tfm-ror51-rubygem-mini_mime/tfm-ror51-rubygem-mini_mime.spec [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+Summary: A lightweight mime type lookup toy
+Group:   Development/Languages
+License: MIT
+URL:     https://github.com/discourse/mini_mime
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygem(minitest) >= 5.0
```
Comment (ekohl) [inline packages/tfm-ror51-rubygem-mini_mime/tfm-ror51-rubygem-mini_mime.spec]: In the template it looks like this is supposed to be skipped.

File: packages/tfm-ror51-rubygem-erubi/tfm-ror51-rubygem-erubi.spec [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+Summary: Small ERB Implementation
+Group:   Development/Languages
+License: MIT
+URL:     https://github.com/jeremyevans/erubi
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygem(minitest)
```
Comment (ekohl) [inline packages/tfm-ror51-rubygem-erubi/tfm-ror51-rubygem-erubi.spec]: The template suggests this should be skipped.

**Comment by evgeni** [comment]
APC, I guess ;)

**Comment by ehelms** [comment]
Fixed those two specs (they were done before I fully edited the template)

**Pull request merged**